packages/regenerate/regenerate.0.2/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+version: "0.2"
+maintainer: "Drup <drupyog@zoho.com>"
+authors: "Drup <drupyog@zoho.com>"
+license: "ISC"
+homepage: "https://github.com/regex-generate/regenerate"
+bug-reports: "https://github.com/regex-generate/regenerate/issues"
+dev-repo: "git+https://github.com/regex-generate/regenerate.git"
+doc: "https://regex-generate.github.io/regenerate/doc/0.2/"
+
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.2"}
```
Dune should not be tagged as a build dependency. Moreover according to your `dune-project` file, this package requires at least dune 1.2